cligj/features.py

```diff
@@ -115,6 +129,26 @@ def iter_features(geojsonfile, func=None):
                     newfeat = func(feat)
                     if newfeat:
                         yield newfeat
+            elif 'coordinates' in obj:
+                newfeat = func(to_feature(obj))
+                if newfeat:
+                    yield newfeat
+
+
+def to_feature(obj):
```

Comment:
@perrygeo I've got no immediate use for what I'm about to suggest: but what about adding a `properties` keyword argument to this function, the value of which would be copied to the result's `properties` member?